data/json/recipes/recipe_deconstruction.json

```diff
@@ -5617,9 +5617,53 @@
     "type": "uncraft",
     "activity_level": "NO_EXERCISE",
     "skill_used": "tailor",
-    "time": "30 s",
-    "components": [ [ [ "string_6", 6 ] ] ],
-    "flags": [ "BLIND_EASY" ]
+    "time": "60 s",
+    "qualities": [ { "id": "CUT", "level": 2 } ],
+    "components": [ [ [ "string_6", 6 ] ] ]
```

Comment:
I believe that you have the components and results backwards for many if not all of these.
This one, for example "deconstructs" 6x 6-inch strings into a single 36-inch string.
Because you start with the components and produce the byproducts and result.

Shouldn't it be:
`"result": "string_6",`
`"byproducts": [ [ "string_6", 5 ] ],`
`"components": [ [ [ "string_36", 1 ] ] ]`
Which would start wit a 36" string and create 6 total 6 inch strings.

Reply:
You'd think so! But that's how an uncraft recipe is formatted. 

If you look at 
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/recipes/recipe_deconstruction.json
all the other entries follow the same logic.

`  {
    "result": "bead_bracelet",
    "type": "uncraft",
    "activity_level": "NO_EXERCISE",
    "time": "6 s",
    "qualities": [ { "id": "CUT", "level": 1 } ],
    "components": [ [ [ "wooden_bead", 10 ] ] ]
  },`
  
  The components are what you get from disassembly.

Reply:
You're right. Good catch!
How very unintuitive.